api/go.mod

```diff
@@ -251,6 +252,7 @@ replace (
 
 	github.com/caraml-dev/turing/engines/experiment => ../engines/experiment
 	github.com/caraml-dev/turing/engines/router => ../engines/router
+	github.com/go-gota/gota => github.com/gojekfarm/gota v0.12.1-0.20230221101638-6cd9260bd598
```

Comment:
Because I got this error while install dependency
<img width="939" alt="image" src="https://github.com/user-attachments/assets/7bd868cc-61b1-4de0-b239-c6f2737a7046">

Reply:
I think this is an error from intellij only, I got this too, idk why tho 😅 but the app can run fine without explicitly replace the `gota` in `go.mod`, I just run `go mod vendor` manually (not using intellij dependencies sync). Does this not work for you?

Reply:
Yeah I seem to get that error too on my IDE 😅 and I also run `go mod vendor` manually 🧐 Not sure why this happens though, but I'll just remove it for the time being unless it causes some problem on the app.